ai/github_issue_handling/usage_en.txt

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Could you follow the github document format and move it to the doc folder? You can refer to Guoyang's PR in https://github.com/XinShuYang/antrea/pull/11